<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Reservations</class>
 <widget class="QWidget" name="Reservations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reservations</string>
  </property>
  <property name="styleSheet">
   <string notr="true">Reservations {
    background-color: #f0f0f0;
    color: #333333;
    font-family: 'Segoe UI', Arial, sans-serif;
    font-size: 13px;
}

QLabel {
    font-size: 14px;
    color: #333333;
}

QPushButton {
    background-color: #70B9FE;
    color: #ffffff;
    border: none;
    border-radius: 5px;
    padding: 8px 15px;
    font-size: 13px;
    min-height: 30px;
    min-width: 80px;
}

QPushButton:hover {
    background-color: #5aa9fe;
}

QPushButton:pressed {
    background-color: #4a99fe;
}

QLineEdit {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 4px;
    padding: 6px;
    font-size: 13px;
    color: #333333;
}

QLineEdit:focus {
    border-color: #70B9FE;
    box-shadow: 0 0 5px rgba(112, 185, 254, 0.5);
}

QLineEdit:read-only {
    background-color: #e6e6e6;
    color: #666666;
}

QComboBox {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 4px;
    padding: 6px;
    font-size: 13px;
    color: #333333;
    min-width: 150px;
}

QComboBox:hover {
    border-color: #70B9FE;
}

QComboBox:focus {
    border-color: #70B9FE;
    box-shadow: 0 0 5px rgba(112, 185, 254, 0.5);
}

QComboBox::drop-down {
    border: none;
    width: 20px;
}

QComboBox QAbstractItemView {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    selection-background-color: #70B9FE;
    selection-color: #ffffff;
    color: #333333;
}

QDateEdit {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 4px;
    padding: 6px;
    font-size: 13px;
    color: #333333;
}

QDateEdit:focus {
    border-color: #70B9FE;
    box-shadow: 0 0 5px rgba(112, 185, 254, 0.5);
}

QDateEdit::drop-down {
    border: none;
    width: 20px;
}

QDoubleSpinBox {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 4px;
    padding: 6px;
    font-size: 13px;
    color: #333333;
    min-width: 100px;
}

QDoubleSpinBox:focus {
    border-color: #70B9FE;
    box-shadow: 0 0 5px rgba(112, 185, 254, 0.5);
}

QTableView {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    border-radius: 4px;
    gridline-color: #e6e6e6;
    font-size: 12px;
    color: #333333;
}

QTableView::item {
    padding: 4px;
}

QTableView::item:selected {
    background-color: #70B9FE;
    color: #ffffff;
}

QHeaderView::section {
    background-color: #e6e6e6;
    border: none;
    padding: 6px;
    font-size: 13px;
    font-weight: bold;
    color: #333333;
}

QHeaderView::section:hover {
    background-color: #d6d6d6;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelUser">
       <property name="text">
        <string>User ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxUser"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelUserName">
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="userNameDisplay">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Selected user's name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelRoom">
       <property name="text">
        <string>Room</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="roomLayout">
       <item>
        <widget class="QComboBox" name="comboBoxRoom"/>
       </item>
       <item>
        <widget class="QLabel" name="labelPreferred">
         <property name="text">
          <string>Preferred:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPreferred">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelCheckIn">
       <property name="text">
        <string>Check-In</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="dateEditCheckIn">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelCheckOut">
       <property name="text">
        <string>Check-Out</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="dateEditCheckOut">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelAmount">
       <property name="text">
        <string>Total Amount</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="spinBoxAmount">
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelPayment">
       <property name="text">
        <string>Payment Status</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="comboBoxPayment">
       <item>
        <property name="text">
         <string>Paid</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pending</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelStatus">
       <property name="text">
        <string>Reservation Status</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="comboBoxStatus">
       <item>
        <property name="text">
         <string>Confirmed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canceled</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUpdate">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRefresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableViewReservations">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
